frontend/src/component/strategies/CustomStrategyInfo/CustomStrategyInfo.tsx

```diff
@@ -17,11 +17,8 @@ export const CustomStrategyInfo: FC<{ alert?: boolean }> = ({ alert }) => {
     const content = (
         <>
             <Paragraph>
-                We recommend you to use the predefined strategies like Gradual
-                rollout with constraints instead of creating a custom strategy.
-            </Paragraph>
-            <Paragraph>
-                If you decide to create a custom strategy be aware of:
+                If you decide to create a custom strategy be aware of the
```

Comment:
The recommendation is handled by the warnings now instead. No point in duplicating the information